### PR TITLE
Added README.md for dummy_sensors

### PR DESCRIPTION
**Pull request by cardboardcode**
## Purpose of Pull Request

This pull request adds a `README.md` to **dummy_sensors** as part of efforts to address the issue highlighted in https://github.com/ros2/demos/issues/531.

### CODE REVIEW

File: dummy_robot/dummy_sensors/README.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+## **What Is This?**
+
+This demo creates and runs 2 ROS2 **nodes** individually, namely `dummy_laser` and `dummy_joint_states`.
```
Comment (clalancette) [inline dummy_robot/dummy_sensors/README.md:3]:
```suggestion
This demo creates and runs two ROS 2 **nodes** individually, namely `dummy_laser` and `dummy_joint_states`.
```

File: dummy_robot/dummy_sensors/README.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+**dummy_laser** publishes `sensor_msgs::msg::LaserScan` on a ROS2 publisher topic, `/scan`.
```
Comment (clalancette) [inline dummy_robot/dummy_sensors/README.md:5]:
```suggestion
**dummy_laser** publishes `sensor_msgs::msg::LaserScan` on a ROS 2 publisher topic, `/scan`.
```

File: dummy_robot/dummy_sensors/README.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+
+**dummy_joint_states** publishes `sensor_msgs::msg::JointState` on a ROS2 publisher topic, `/joint_states`.
```
Comment (clalancette) [inline dummy_robot/dummy_sensors/README.md:7]:
```suggestion
**dummy_joint_states** publishes `sensor_msgs::msg::JointState` on a ROS 2 publisher topic, `/joint_states`.
```

File: dummy_robot/dummy_sensors/README.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+
+Both **dummy_laser** and **dummy_joint_states** are dependencies of **dummy_robot_bringup**. Please refer to [dummy_robot_bringup](https://github.com/ros2/demos/tree/rolling/dummy_robot/dummy_robot_bringup).
```
Comment (clalancette) [inline dummy_robot/dummy_sensors/README.md:9]:
```suggestion
Both **dummy_laser** and **dummy_joint_states** are dependencies of **dummy_robot_bringup**, and are usually expected to be run from the launch file there.
Please refer to [dummy_robot_bringup](https://github.com/ros2/demos/tree/rolling/dummy_robot/dummy_robot_bringup).
```

**Review by clalancette — CHANGES_REQUESTED** [review]

File: dummy_robot/dummy_sensors/README.md (hunk 1 above)
Comment (cardboardcode) [reply dummy_robot/dummy_sensors/README.md:3]: Noted. Will edit accordingly.

File: dummy_robot/dummy_sensors/README.md (hunk 2 above)
Comment (cardboardcode) [reply dummy_robot/dummy_sensors/README.md:5]: Done.

File: dummy_robot/dummy_sensors/README.md (hunk 3 above)
Comment (cardboardcode) [reply dummy_robot/dummy_sensors/README.md:7]: Done.

File: dummy_robot/dummy_sensors/README.md (hunk 4 above)
Comment (cardboardcode) [reply dummy_robot/dummy_sensors/README.md:9]: Done.

File: dummy_robot/dummy_sensors/README.md [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+velocity: []
+effort: []
+---
+```
+
+
+## **References**
+
+- `sensor_msgs::msg::LaserScan` Message Format: https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/LaserScan.msg
+- `sensor_msgs::msg::JointState` Message Format: https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/JointState.msg
```
Comment (clalancette) [inline dummy_robot/dummy_sensors/README.md:165]:
```suggestion
- `sensor_msgs/msg/LaserScan` Message Format: https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/LaserScan.msg
- `sensor_msgs/msg/JointState` Message Format: https://github.com/ros2/common_interfaces/blob/rolling/sensor_msgs/msg/JointState.msg
```

(using slashes is kind of our language-agnostic way to refer to messages)

File: dummy_robot/dummy_sensors/README.md [hunk 6]
```diff
@@ -0,0 +1,165 @@
+## **What Is This?**
+
+This demo creates and runs two ROS 2 **nodes** individually, namely `dummy_laser` and `dummy_joint_states`.
+
+**dummy_laser** publishes `sensor_msgs::msg::LaserScan` on a ROS 2 publisher topic, `/scan`.
```
Comment (clalancette) [inline dummy_robot/dummy_sensors/README.md:5]:
```suggestion
**dummy_laser** publishes `sensor_msgs/msg/LaserScan` on a ROS 2 publisher topic, `/scan`.
```

File: dummy_robot/dummy_sensors/README.md [hunk 7]
```diff
@@ -0,0 +1,165 @@
...
+
+**dummy_joint_states** publishes `sensor_msgs::msg::JointState` on a ROS 2 publisher topic, `/joint_states`.
```
Comment (clalancette) [inline dummy_robot/dummy_sensors/README.md:7]:
```suggestion
**dummy_joint_states** publishes `sensor_msgs/msg/JointState` on a ROS 2 publisher topic, `/joint_states`.
```

**Review by clalancette — CHANGES_REQUESTED** [review]
Sorry, a couple more changes that I missed last time.  With that done, this looks good to me.  Thanks for iterating.

File: dummy_robot/dummy_sensors/README.md (hunk 5 above)
Comment (cardboardcode) [reply dummy_robot/dummy_sensors/README.md:165]: Noted. Learnt something new today. Nice. Done.

File: dummy_robot/dummy_sensors/README.md (hunk 6 above)
Comment (cardboardcode) [reply dummy_robot/dummy_sensors/README.md:5]: Done.

**Pull request merged**